src/test/java/core/basesyntax/dao/impl/FruitDaoImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.dao.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.db.FruitStorage;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private static final String APPLE = "apple";
+    private static final String BANANA = "banana";
+    private static final int DEFAULT_QUANTITY = 50;
+    private static final int NEW_QUANTITY = 100;
+    private static FruitDao fruitDao;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitDao = new FruitDaoImpl();
+    }
+
+    @Before
+    public void beforeEach() {
+        FruitStorage.fruitStorage.put(APPLE, DEFAULT_QUANTITY);
+    }
+
+    @After
+    public void afterEach() {
```

Comment:
```suggestion
    public void clearStorage() {
```
 the similar naming should be in other test classes, don't forget, name of the methods should start with verb